docs/content/Examples.md

```diff
@@ -6,86 +6,77 @@ redirect_from:
+
+The following tutorials cover advanced concepts of Cube.js:
+
+* [Pre-Aggregations Tutorial](https://cube.dev/blog/high-performance-data-analytics-with-cubejs-pre-aggregations/) -
+  you will learn about pre-aggregations, one of the most powerful Cube.js features that significantly speed up the performance of your applications such as dashboards and reports
+
+* *Building an Open Source Mixpanel Alternative* -
+  a two-part series on building a production-ready application with Cube.js:
+  [Part 1: Collecting and Displaying Events](https://cube.dev/blog/building-an-open-source-mixpanel-alternative-1),
+  [Part 2: Conversion Funnels ](https://cube.dev/blog/building-open-source-mixpanel-alternative-2/)
+
+
+### Features
+
+Learn more about prominent features of Cube.js:
+
+| Feature                                                                                  | Story                                                                                                                | Demo                                              |
+| :--------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------ |
+| [Drill downs](https://cube.dev/docs/drill-downs)                                         | [Introducing a drill down table API](https://cube.dev/blog/introducing-a-drill-down-table-api-in-cubejs/)            | [Demo](https://drill-downs-demo.cube.dev)         |
+| [Compare date range](https://cube.dev/docs/query-format#time-dimensions-format)          | [Comparing data over different time periods](https://cube.dev/blog/comparing-data-over-different-time-periods/)      | [Demo](https://compare-date-range-demo.cube.dev)  |
+| [Data blending](https://cube.dev/docs/data-blending)                                     | [Introducing data blending API](https://cube.dev/blog/introducing-data-blending-api/)                                | [Demo](https://data-blending-demo.cube.dev)       |
+| [Real-time data fetch](https://cube.dev/docs/real-time-data-fetch)                       | [Real-time dashboard guide](https://real-time-dashboard.cube.dev)                                                    | [Demo](https://real-time-dashboard-demo.cube.dev) |
+| [Dynamic schema creation](https://cube.dev/docs/dynamic-schema-creation)                 | [Using asyncModule to generate schemas](https://github.com/cube-js/cube.js/tree/master/examples/async-module-simple) | —                                                 |
+| [JSON Web Ket Sets support](http://localhost:8000/security#using-json-web-key-sets-jwks) | [Auth0 integration](https://github.com/cube-js/cube.js/tree/master/examples/auth0)                                   | —                                                 |
```

Comment:
Typo:

```suggestion
| [JSON Web Key Sets support](http://localhost:8000/security#using-json-web-key-sets-jwks) | [Auth0 integration](https://github.com/cube-js/cube.js/tree/master/examples/auth0)                                   | —                                                 |
```

Also wondering if the feature name should be Authentication, to match the docs? /cc @keydunov